test/unit/controllers/declaration/AdditionalActorsAddControllerSpec.scala

```diff
@@ -119,7 +119,7 @@ class AdditionalActorsAddControllerSpec extends ControllerSpec with ErrorHandler
       "user put incorrect data" in {
 
         val longerEori = TestHelper.createRandomAlphanumericString(18)
-        val wrongAction = Seq(("eori", longerEori), ("partyType", "CS"), saveAndContinueActionUrlEncoded)
+        val wrongAction = Seq(("eoriCS", longerEori), ("partyType", "CS"), saveAndContinueActionUrlEncoded)
```

Comment:
Why you changed this?

Reply:
Because the test was passing incorrectly.  It was returning a 400 because the eori field was incorrectly named, not because the controller found anything wrong with the data (i.e. duplicate or value too long, etc)

Reply:
Yeah, got it, this page has field name for eori with party type.

Thanks